src/widget/wcolorpicker.cpp

```diff
@@ -12,7 +12,7 @@ namespace {
     const int kNumColumns = 4;
 }
 
-WColorPicker::WColorPicker(QWidget* parent)
+WColorPicker::WColorPicker(QWidget* parent, bool allowNoColor)
```

Comment:
`bool allowNoColor` could be replaced by an `enum class` with 2 enum variants to improve type-safety.

Reply:
Do you think that's better than just using a bool when there can only be two values? The current implementation seems more ideomatic to me.

Reply:
The meaning of passing false or true is unclear at the call side without the parameter name.

Reply:
Added in 49bd30ea7fc768675961e6f2c2f2656acffa4698.